### PR TITLE
Fix to run tests

### PR DESCRIPTION
**Pull request by jeffb-sfdc**
This PR address a few issues.  As a new user to `wdio-vscode-service` I ran into a few stumbling blocks while getting up and running, and I think this PR will benefit others.  This PR includes:

* Added a launch.json file, so one can debug the tests.
* Added a "Contributing" section to README.md, so beginners have a set of instructions on getting started.
* Removed the `grep` property from `mochaOpts`.  When I ran, I found that the tests weren't actually running.  (see https://github.com/webdriverio-community/wdio-vscode-service/issues/41). Removing this option fixes the issue.
* Added some fixes to basic.e2e.ts.  After I got the tests to run, I found there were two tests which were failing, and so I added fixes for the tests as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -320,6 +320,16 @@ If you use WebdriverIO with TypeScript make sure to add `wdio-vscode-service` to
...
 
+## Contributing
+
+Before posting a pull request, please run the following:
+
+1. clone wdio-vscode-service
+2. cd wdio-vscode-service
+3. npm install
+4. npm run build
+5. npm run test (or npm run ci)
```
Comment (christian-bromann) [inline README.md:331]:
```suggestion
1. `git clone git@github.com:webdriverio-community/wdio-vscode-service.git`
1. `cd wdio-vscode-service`
1. `npm install`
1. `npm run build`
1. `npm run test (or npm run ci)`
```

**Review by christian-bromann — APPROVED** [review]
Awesome, this looks good to me 👍

**Comment by christian-bromann** [comment]
It seems like there are some issues with Chromedriver setup. I am currently on vacation and can check back next week.

File: README.md [hunk 2]
```diff
@@ -324,11 +324,11 @@ If you use WebdriverIO with TypeScript make sure to add `wdio-vscode-service` to
...
-1. clone wdio-vscode-service
-2. cd wdio-vscode-service
-3. npm install
-4. npm run build
-5. npm run test (or npm run ci)
+1. `git clone git@github.com:webdriverio-community/wdio-vscode-service.git`
+1. `cd wdio-vscode-service`
+1. `npm install`
+1. `npm run build`
+1. `npm run test (or npm run ci)`
```
Comment (jeffb-sfdc) [inline README.md:331]: Cool, looks good.

**Comment by jeffb-sfdc** [comment]
> 

Thanks @christian-bromann , yea, let's pick up next week.  It looks like my PR is now failing in CI, and is failing due to the presence of `console.log()`, however I didn't add any logging, and I didn't turn on this lint setting.  Anyway, let's revisit next week, thanks!

**Comment by christian-bromann** [comment]
Once #45 is merged we should rebase this PR on top of `main`.

**Comment by jeffb-sfdc** [comment]
> 
Cool, sounds good @christian-bromann

**Comment by christian-bromann** [comment]
@jeffb-sfdc mind updating the branch and rebase it on top of `main`?

**Comment by jeffb-sfdc** [comment]
> 

Done, though please look over the changes.  I ran into issues resolving the conflicts using GutHub so I resolved them by hand.

**Review by christian-bromann — APPROVED** [review]
Great work 🎉  thank you!

**Pull request merged**